Hello, 

I'm the maintainer of wally-cli, and would love to help getting this to work.

The lint error is

./dfu.go:180:25: Sprintf call has arguments but no formatting directives

There's no Sprintf directive on line 180 https://github.com/zsa/wally-cli/blob/75e758546d0679f4ce5b562625e3de21b45561f1/dfu.go#L180

There's one on line 181, and it does have a formatting directive.

Regarding the version bump, no worries I'll tag a version once we get that linting issue sorted out.

--- a/pkgs/development/tools/wally-cli/default.nix
+++ b/pkgs/development/tools/wally-cli/default.nix
@@ -2,7 +2,7 @@
 
 buildGoModule rec {
   pname = "wally-cli";
-  version = "2.0.0";
+  version = "2.0.1";
 
   subPackages = [ "." ];
 
@@ -14,14 +14,11 @@ buildGoModule rec {
     owner = "zsa";
     repo = "wally-cli";
     rev = "${version}-linux";
-    sha256 = "0xz3z18bbnf736ngjj6jhnp3p2j55m5jhnb2xl6l5hybracfyhm7";
+    sha256 = "NuyQHEygy4LNqLtrpdwfCR+fNy3ZUxOClVdRen6AXMc=";
   };
 
-  vendorSha256 = "0jqx38x5qvir6zc5yq9p2adafwqhy4hil1k5g81rr1fvbn06k3a6";
+  vendorSha256 = "AVYG+aLpAXohUOORV/uPw7vro+Kg98+AmSmYGHtOals=";
   runVend = true;
-
-  # Can be removed when https://github.com/zsa/wally-cli/pull/1 is merged.
-  doCheck = false;
 
   meta = with lib; {
     description = "A tool to flash firmware to mechanical keyboards";

